Tests/ReactiveSwiftTests/SignalSpec.swift

```diff
@@ -213,6 +214,96 @@ class SignalSpec: QuickSpec {
 			}
 		}
 
+		describe("interruption") {
+			it("should not send events after sending an interrupted event") {
+				let queue: DispatchQueue
+				let counter = Atomic<Int>(0)
+
+				if #available(macOS 10.10, *) {
+					queue = DispatchQueue.global(qos: .userInitiated)
+				} else {
+					queue = DispatchQueue.global(priority: .high)
+				}
+
+				let (signal, observer) = Signal<Int, NoError>.pipe()
+
+				var hasSlept = false
+				var events = [Event<Int, NoError>]()
+
+				let sema = DispatchSemaphore(value: 0)
+
+				signal.observe { event in
+					if !hasSlept {
+						sema.signal()
+						sleep(5)
```

Comment:
Does this artificially add 5 seconds to the test? If so, 👎 that's too high of a price to pay.